Added warning that disabling auditing impacts the platform (#4333)
Particular/docs.particular.net: nservicebus/operations/auditing.md

 - nservicebus/auditing-with-nservicebus
---

> WARNING: Disabling auditing impacts the functionality of the platform tools. ServicePulse can work without auditing enabled but will show incorrect retry states on messages, and ServiceInsight will only show failures instead of complete conversations.

Reviewer: ```suggestion
NOTE: Auditing is required for optimum functionality of the platform tools. ServicePulse can work without auditing enabled but will show incorrect retry states on messages, and ServiceInsight will only show failures instead of complete conversations.
```

I would also put this below before Message headers, somewhere near line 24, instead of at the top. Enabling auditing also has a performance impact on ServiceControl which sometimes means we recommend not auditing some of your endpoints.
